training/hc-leipzig-7t-mp2rage/plot_nnunet_different_training_logs.py

```diff
@@ -0,0 +1,311 @@
+"""
+This script makes a visual comparison of the validation pseudo dice across epochs for multiple nnUNet training logs.
+The script reads the log files, extracts the epoch number and validation pseudo dice, and plots them.
+
+NOTE: This script is a modified version of
+https://github.com/ivadomed/utilities/blob/main/training_scripts/plot_nnunet_training_log.py script, where the
+functionality was extended to compare multiple training logs in specific spinal level.
+
+Usage:
+    python plot_nnunet_different_training_logs.py -i <path_to_log_file(s)> -spinal-level <spinal_level>
+    -output-image <output_image>
+"""
+
+import os
+import re
+import argparse
+import pandas as pd
+import seaborn as sns
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def get_parser():
+    """
+    parser function
+    """
+
+    parser = argparse.ArgumentParser(
+        description='Plot epoch number vs Pseudo dice based on a nnUNet training log file for multiple logs '
+                    '- you can choose the spinal level for visualization.',
+        prog=os.path.basename(__file__).strip('.py')
+    )
+    parser.add_argument(
+        '-i',
+        required=True,
+        nargs='+',
+        type=str,
+        help='Path to the txt log file produced by nnUNet (or more log files separated by space).'
+             'Example: training_log_2024_1_22_11_09_18.txt'
+    )
+    parser.add_argument(
+        '-spinal-level',
+        required=True,
+        nargs='+',
+        type=str,
+        help='Spinal level of the data for comparison training logs.'
```

Comment:
Does the script expect `2` or `C2`? Please clarify

```suggestion
        help='Spinal level to plot. Example: XX'
```